ksql-engine/src/main/java/io/confluent/ksql/exception/KafkaDeleteTopicsException.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.exception;
+
+import java.util.List;
+import javafx.util.Pair;
+
+public class KafkaDeleteTopicsException extends KafkaTopicClientException {
```

Comment:
I'm confused. It looks like we now throw this Exception from `KafkaTopicCLientImpl` but we don't check for it anywhere (other than unit tests). Do we still need it? If all you want to do is have the ability to throw multiple exceptions, use `Exception#addSuppressed`

Reply:
I think the thought was that there could be use for this in the future if when we delete multiple topics at once and there are multiple errors, the details for each topic's deletion error would be available. For example, in ClusterTerminator.java, the error message given in that files deleteTopics() function is very vague and only lists the topic names.

`throw new KsqlException("Exception while deleting topics: " + StringUtils.join(topicsToBeDeleted, ", "));`

Reply:
I think this is a perfect use case for `Exception#addSuppressed `, but that's personal style so feel free to choose what you like more.